astropy/coordinates/sky_coordinate.py

```diff
@@ -1027,18 +1027,23 @@ def position_angle(self, other):
 
         return angle_utilities.position_angle(slon, slat, olon, olat)
 
-    def astrometric_frame(self):
+    def astrometric_frame(self, rotation=None):
         """
         Returns the astrometric frame with this `SkyCoord` at the origin.
 
         Returns
         -------
-        astrframe: AstrometricFrame
+        astrframe : `~astropy.coordinates.AstrometricFrame`
             An astrometric frame of the same type as this `SkyCoord` (e.g., if
             this object has an ICRS coordinate, the resulting frame is
             AstrometricFrame with an ICRS origin)
+        rotation : Quantity with angle units
```

Comment:
I copy & pasted this from the `AstrometricFrame` docstring. Do you want me to change it in both places, just here, or leave as is?